index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Microverse</title>
+    <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+    <h3 class="header">Hello Microverse!</h3>
+</body>
```

Comment:
- [ ] Nice work! But it will be better if you implement it as it is in the requirement
![image](https://user-images.githubusercontent.com/2903437/127062055-e6906ba8-9f3e-43b0-9a4c-baee57eb42c0.png)
- You will need to include a header element and an h1 tag. It is good to use semantics HTML for best standard industry practices.